_includes/footer.html

```diff
@@ -3,7 +3,6 @@
     <div class="column d-4-12 m-2-12">
       <h4>For Competitors</h4>
       <ul class="sitemap">
-        <li><a href="{{ '/teams/' | prepend: site.baseurl }}">Teams Dashboard</a></li>
         <li><a href="https://studentrobotics.org/docs/">Documentation</a></li>
```

Comment:
Perhaps we should also include here a link to the events pages? Almost all the events we post are going to be for the competitors, yet the list of events has become increasingly hard to find since the new layout.

Reply:
We're intending to move at least a summary to the top of the homepage to address this https://github.com/srobo/marketing-team-minutes/issues/28

Reply:
oh just realised this was the footer - yes that's a good idea